src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.types.ts

```diff
@@ -16,7 +16,7 @@ export interface IAdvancedSearchResultDetailsListProps {
     containsError: boolean;
     isLoading: boolean;
     /* Callback function from parent on what to do once the user selects or deselects a twin. */
-    onTwinSelection: (IADTTwin) => void;
+    onTwinIdSelect: (twinId: string) => void;
```

Comment:
nit: `onRowSelect`? or `onSelectRow`

Reply:
If it where generic I would change it to any of those, but I prefer to specify what is being selected.